client/my-sites/people/people-profile/index.jsx

```diff
@@ -254,4 +273,10 @@ class PeopleProfile extends React.PureComponent {
 	}
 }
 
-export default localize( PeopleProfile );
+export default connect( ( _state, { siteId, user } ) => {
+	const userId = user && user.ID;
+	const externalContributors = ( siteId && requestExternalContributors( siteId ).data ) || [];
```

Comment:
noting our lack of a pending or failure state here - not sure if it matters

we might end up making it look like someone _isn't_ an external contributor even though they are because we aren't showing that the data hasn't loaded yet or that it failed to load